system-contracts/bootloader/test_infra/src/main.rs

```diff
@@ -172,13 +173,16 @@ fn execute_internal_bootloader_test() {
                         "Should have failed with {}, but run successfully.",
                         requested_assert
                     )),
-                    ExecutionResult::Revert { output } => Err(format!(
+                    ExecutionResult::Revert { output } => {
+                        println!("OUTPUT: {:?}", output);
+                        Err(format!(
                         "Should have failed with {}, but run reverted with {}.",
                         requested_assert,
                         output.to_user_friendly_string()
-                    )),
+                    ))},
                     ExecutionResult::Halt { reason } => {
                         if let Halt::UnexpectedVMBehavior(reason) = reason {
+                            println!("{:?}", reason);
```

Comment:
we dont have any error logs atm? Trying to understand the difference between the new println and the old returned `Err`

Reply:
I think this might have mostly been for debugging purposes, we can and should remove this safely.